### ISSUE TITLE
some problems with the use of the software.

### ISSUE THREAD

**Issue by BlackCullen** [iholston/lol-bot#277]
In gui, LOL directory does not support Chinese, will be garbled, player names and heroes do not show, does not support utf-8, can you add language pack configuration, you can directly translate the software, and the hero name configuration file, you can also translate the hero name.
Add hero selection, which route options, for example, the upper road, the middle road, the lower road.
Character combat logic is relatively simple, the number of deaths is relatively high, for example: chasing people into the tower, do not avoid, and do not push the tower. Not very smart, I originally used a script that used a shooter to hit the entry man with random random kills and had a good record. That script logic was to use shooters.
Thanks for writing the script!

![1](https://github.com/user-attachments/assets/d116b7f0-3bc8-4dfc-9587-36dbd61151fe)
![2](https://github.com/user-attachments/assets/013e8724-46a8-4279-b809-f8391316bbe7)

**Comment by iholston** [comment]
Does the bot still run even though the language is not supported?

**Comment by BlackCullen** [comment]
> 即使不支持该语言，机器人还能运行吗？

Yes, Biggie. The game directory is changed to English and it runs, the fist api is all universal. I'm coding it here and have some issues.
The battle logic is rather dull, always giving away heads and not pushing towers. Below is the battle log I've played with the chart color script.
Sometimes there are problems with the window configurations, can't save, always full screen, always change your window configurations, and you will go full screen even if you set it up successfully.

Translated with DeepL.com (free version)

**Comment by cjz9032** [comment]
Wow ，greate, thanks ，but just curious at nowdays still project does not pick utf8 as default string encoding?

**Comment by cjz9032** [comment]
> 
> In gui, LOL directory does not support Chinese, will be garbled, player names and heroes do not show, does not support utf-8, can you add language pack configuration, you can directly translate the software, and the hero name configuration file, you can also translate the hero name.
> Add hero selection, which route options, for example, the upper road, the middle road, the lower road.
> Character combat logic is relatively simple, the number of deaths is relatively high, for example: chasing people into the tower, do not avoid, and do not push the tower. Not very smart, I originally used a script that used a shooter to hit the entry man with random random kills and had a good record. That script logic was to use shooters.
> Thanks for writing the script!
> 
> ![1](https://github.com/user-attachments/assets/d116b7f0-3bc8-4dfc-9587-36dbd61151fe)
> ![2](https://github.com/user-attachments/assets/013e8724-46a8-4279-b809-f8391316bbe7)
> 

哥们 windows跑起来稳吗，我还想装个黑苹果跑的

**Comment by BlackCullen** [comment]
> 哥们windows跑起来稳定吗，我还想装个黑苹果跑的
这里没什么问题，原来的版本没有被封过，有ACE的版本也试过几次。就是依赖队友赢。毕竟没有那么智能

**Comment by BlackCullen** [comment]
> > 哥们windows跑起来稳定吗，我还想装个黑苹果跑的
> > 这里没什么问题，原来的版本没有被封过，有ACE的版本也试过几次。就是依赖时钟赢了。毕竟没有那么智能



> > 在gui中，LOL目录不支持中文，会乱码，玩家名和英雄不显示，不支持utf-8，能不能加语言包配置，可以直接翻译软件，还有英雄名配置文件，也可以翻译英雄名。
> > 增加英雄选择，哪些路线选项，比如说上路，中路，下路。
> > 人物战斗逻辑比较简单，死亡次数比较多，比如说：追人进塔，不躲避，不推塔。不是很智能，我本来用的一个脚本是用射手随机打进塔的，战绩不错。那个脚本逻辑就是用射手。
> > 感谢写脚本！
> > ![1](https://github.com/user-attachments/assets/d116b7f0-3bc8-4dfc-9587-36dbd61151fe)
> > ![2](https://github.com/user-attachments/assets/013e8724-46a8-4279-b809-f8391316bbe7)
> 
> 哥们windows跑起来稳定吗，我还想装个黑苹果跑的

哥们windows跑起来稳定吗，我还想装个黑苹果跑的
这里没什么问题，原来的版本没有被封过，有ACE的版本也试过几次。就是依赖时钟赢了。毕竟没有那么智能

**Comment by BlackCullen** [comment]
> > 在gui中，LOL目录不支持中文，会乱码，玩家名和英雄不显示，不支持utf-8，能不能加语言包配置，可以直接翻译软件，还有英雄名配置文件，也可以翻译英雄名。
> > 增加英雄选择，哪些路线选项，比如说上路，中路，下路。
> > 人物战斗逻辑比较简单，死亡次数比较多，比如说：追人进塔，不躲避，不推塔。不是很智能，我本来用的一个脚本是用射手随机打进塔的，战绩不错。那个脚本逻辑就是用射手。
> > 感谢写脚本！
> > ![1](https://github.com/user-attachments/assets/d116b7f0-3bc8-4dfc-9587-36dbd61151fe)
> > ![2](https://github.com/user-attachments/assets/013e8724-46a8-4279-b809-f8391316bbe7)
> 
> 哥们windows跑起来稳定吗，我还想装个黑苹果跑的
我在win平台和VM中跑起来没什么问题，除了不智能，赢的话比较依赖队友还可以.

**Comment by cjz9032** [comment]
> > > 在gui中，LOL目录不支持中文，会乱码，玩家名和英雄不显示，不支持utf-8，能不能加语言包配置，可以直接翻译软件，还有英雄名配置文件，也可以翻译英雄名。
> > > 增加英雄选择，哪些路线选项，比如说上路，中路，下路。
> > > 人物战斗逻辑比较简单，死亡次数比较多，比如说：追人进塔，不躲避，不推塔。不是很智能，我本来用的一个脚本是用射手随机打进塔的，战绩不错。那个脚本逻辑就是用射手。
> > > 感谢写脚本！
> > > ![1](https://github.com/user-attachments/assets/d116b7f0-3bc8-4dfc-9587-36dbd61151fe)
> > > ![2](https://github.com/user-attachments/assets/013e8724-46a8-4279-b809-f8391316bbe7)
> > 
> > 
> > 哥们windows跑起来稳定吗，我还想装个黑苹果跑的
> > 我在win平台和VM中跑起来没什么问题，除了不智能，赢的话，比较依赖队友。

后来我改了下, 选VN再激进一点, 基本都能赢了, 因为VN最不依赖技能

**Comment by BlackCullen** [comment]
厉害，我是不会开发。要不这些功能都自己加了



---Original---
From: ***@***.***&gt;
Date: Fri, Mar 21, 2025 20:37 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [iholston/lol-bot] some problems with the use of the software.(Issue #277)


      
在gui中，LOL目录不支持中文，会乱码，玩家名和英雄不显示，不支持utf-8，能不能加语言包配置，可以直接翻译软件，还有英雄名配置文件，也可以翻译英雄名。
 增加英雄选择，哪些路线选项，比如说上路，中路，下路。
 人物战斗逻辑比较简单，死亡次数比较多，比如说：追人进塔，不躲避，不推塔。不是很智能，我本来用的一个脚本是用射手随机打进塔的，战绩不错。那个脚本逻辑就是用射手。
 感谢写脚本！
 1.png (view on web)
 2.png (view on web)
  
哥们windows跑起来稳定吗，我还想装个黑苹果跑的
 我在win平台和VM中跑起来没什么问题，除了不智能，赢的话，比较依赖队友。
   
后来我改了下, 选VN再激进一点, 基本都能赢了, 因为VN最不依赖技能

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID: ***@***.***&gt;
  cjz9032 left a comment (iholston/lol-bot#277)
    
在gui中，LOL目录不支持中文，会乱码，玩家名和英雄不显示，不支持utf-8，能不能加语言包配置，可以直接翻译软件，还有英雄名配置文件，也可以翻译英雄名。
 增加英雄选择，哪些路线选项，比如说上路，中路，下路。
 人物战斗逻辑比较简单，死亡次数比较多，比如说：追人进塔，不躲避，不推塔。不是很智能，我本来用的一个脚本是用射手随机打进塔的，战绩不错。那个脚本逻辑就是用射手。
 感谢写脚本！
 1.png (view on web)
 2.png (view on web)
  
哥们windows跑起来稳定吗，我还想装个黑苹果跑的
 我在win平台和VM中跑起来没什么问题，除了不智能，赢的话，比较依赖队友。
   
后来我改了下, 选VN再激进一点, 基本都能赢了, 因为VN最不依赖技能
